'make get' returns an error 'pub: not found'

When I run `make get`, I get an error:

```
/bin/sh: 1: pub: not found
Makefile:10 recipe for target 'get' failed
make: *** [get] Error 127
```

This fixes the issue for me:

```diff
diff --git a/Makefile b/Makefile
index 0d372c4..b9585bb 100644
--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ default: book clox jlox
 
 # Run pub get on tool directory.
 get:
-	@ cd ./tool; pub get
+	@ cd ./tool; dart pub get
 
 # Remove all build outputs and intermediate files.
 clean:
```

I'm using Dart 2.10.5.

--- a/Makefile
+++ b/Makefile
@@ -5,9 +5,9 @@ TEST_SNAPSHOT := $(BUILD_DIR)/test.dart.snapshot
 
 default: book clox jlox
 
-# Run pub get on tool directory.
+# Run dart pub get on tool directory.
 get:
-	@ cd ./tool; pub get
+	@ cd ./tool; dart pub get
 
 # Remove all build outputs and intermediate files.
 clean:

